code/_global_vars/lists/names.dm

```diff
@@ -13,3 +13,16 @@ var/global/list/verbs =              file2list("config/names/verbs.txt")
 var/global/list/adjectives =         file2list("config/names/adjectives.txt")
 //loaded on startup because of "
 //would include in rsc if ' was used
+
+var/global/list/forbidden_names = list(
```

Comment:
This seems a bit pointless on our codebase.

Reply:
Was this comment addressed?